docs/index.asciidoc

```diff
@@ -21,6 +21,20 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
+If you plan to use the Kibana web interface to analyze data transformed by
+Logstash, use the Elasticsearch output plugin to get your data into
+Elasticsearch. 
+
+This output only speaks the HTTP protocol as it is the preferred protocol for
+interacting with Elasticsearch. In previous versions it was possible to
+communicate with Elasticsearch through the transport protocol, which is now
+reserved for internal cluster communication between nodes
+https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-transport.html.
+This has been deprecated in Elasticsearch 7.0.0 and will be removed in 8.0.0
```

Comment:
@jsvd Thanks for adding this info. Can you clarify this?
"This" (in "This has been deprecated...") means  "transport protocol?"

Reply:
Using the transport protocol to communicate with the cluster

Reply:
Thanks for clarifying. Please see what you think.